### DISCUSSION TITLE
Emoji search feature

### DISCUSSION THREAD

**Discussion by Uniquenameihave** [FossifyOrg/General-Discussion#633] [Ideas]
I request the developer to add the  emoji search feature like in gboard in this fork of simple keyboard

**Comment by Aga-C** [comment]
Can you create a feature request here: https://github.com/FossifyOrg/Keyboard/issues/new?assignees=&labels=feature+request%2C+needs+triage&projects=&template=feature_request.md&title=? Please follow the steps in the template and fill it correctly.

Thanks!
